src/main/groovy/org/jenkinsci/gradle/plugins/jpi/LicenseTask.groovy

```diff
@@ -0,0 +1,94 @@
+package org.jenkinsci.gradle.plugins.jpi
+
+import groovy.xml.MarkupBuilder
+import groovy.xml.QName
+import org.gradle.api.DefaultTask
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.ModuleVersionIdentifier
+import org.gradle.api.artifacts.ResolvedArtifact
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.component.local.model.DefaultProjectComponentIdentifier
+
+class LicenseTask extends DefaultTask {
+    static final String TASK_NAME = 'generateLicenseInfo'
```

Comment:
I think the task name constant should be part of the plugin. If anyone wanted to add a separate `LicenseTask`, then it would probably have a different task name.